### PR TITLE
Updated HPC-X version

### PR DESCRIPTION
**Pull request by artemry-mlnx**

### CODE REVIEW

**Comment by artemry-mlnx** [comment]
CC @Sergei-Lebedev

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI status: <b>PASS</b#  > (click for details)</summary>

**Note:** the artefacts will be deleted after 23-Jun-2020

Artefact file | Status
-- | --
[build_nccl_rdma_sharp_plugins.log](https://gist.githubusercontent.com/mellanox-github/49e40f02340b330ca635670c52c839ca/raw/9bad3371102819d2bfc4175082c112a2c91b001e/build_nccl_rdma_sharp_plugins.log) | :heavy_check_mark: PASS
[configure_sharp_startup.log](https://gist.githubusercontent.com/mellanox-github/49e40f02340b330ca635670c52c839ca/raw/87c6fbaf53e62a61405a1687030992812c2c8721/configure_sharp_startup.log) | :heavy_check_mark: PASS
[configure_sharp_stop.log](https://gist.githubusercontent.com/mellanox-github/49e40f02340b330ca635670c52c839ca/raw/064cf2eb29cb97906791b42997737c40e659965f/configure_sharp_stop.log) | :heavy_check_mark: PASS
[run_nccl_test.log](https://gist.githubusercontent.com/mellanox-github/49e40f02340b330ca635670c52c839ca/raw/f165602783aedb0f88df2f5f0876cb97d4c8bc43/run_nccl_test.log) | :heavy_check_mark: PASS

**Comment by mellanox-github** [comment]
Mellanox CI PASSed.

File: .ci/settings.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ module load "dev/cuda${CUDA_VER}"
 
 # TODO remove use HPC-X which is already inside the image
 #
-HPCX_UBUNTU_INSTALL_DIR=${HPCX_UBUNTU_INSTALL_DIR:-/.autodirect/mtrswgwork/artemry/ci_tools_do_not_remove/hpcx-v2.6.0-gcc-MLNX_OFED_LINUX-5.0-1.0.0.0-ubuntu18.04-x86_64/}
+HPCX_UBUNTU_INSTALL_DIR=${HPCX_UBUNTU_INSTALL_DIR:-/.autodirect/mtrswgwork/artemry/ci_tools_do_not_remove/hpcx-v2.7.pre-gcc-MLNX_OFED_LINUX-5.0-1.0.0.0-ubuntu18.04-x86_64/}
```
Comment (mike-dubman) [inline .ci/settings.sh:25]: should not reside in personal dir but in project space or official location

**Comment by artemry-mlnx** [comment]
To be reworked.

**Pull request closed without merging**